Makefile.include

```diff
@@ -10,6 +10,13 @@ RIOTCPU := $(abspath $(RIOTCPU))
 RIOTBOARD ?= $(RIOTBASE)/boards
 RIOTBOARD := $(abspath $(RIOTBOARD))
 
+# Top level git root of the project being built
+RIOTPROJECT ?= $(shell git rev-parse --show-toplevel)
+RIOTPROJECT := $(abspath $(RIOTPROJECT))
```

Comment:
We ran into problems because some weird people actually chose to use our "release" tarball.